api/src/main/java/org/pmiops/workbench/workspaces/resources/WorkspaceResourceMapper.java

```diff
@@ -135,7 +137,15 @@ default WorkspaceResource fromDbUserRecentlyModifiedResource(
       CohortService cohortService,
       CohortReviewService cohortReviewService,
       ConceptSetService conceptSetService,
-      DataSetService dataSetService) {
+      DataSetService dataSetService,
+      CloudStorageClient cloudStorageClient,
+      WorkspaceAuthService workspaceAuthService) {
+    Set<String> workspaceUsers =
```

Comment:
What do you think about passing `workspaceUsers` as a param here instead of `workspaceAuthService`?

Reply:
I assumed the convention is to send the required services to mapper from controller and will mapper will update the data and set it accordingly.

Reply:
I don't know if the team really has a clear convention for passing parameters to mappers.

The reason I prefer passing `workspaceUsers` here instead of the service is because it's a simpler and smaller thing to pass, which I generally prefer for any function I'm calling.  Also, the only purpose of passing the service here is to generate `workspaceUsers`, which the caller can do just as easily before the call.

The logic with the other services is a little more complicated, which is why I think I went with the other approach.